lib/unifex/specs.ex

```diff
@@ -5,6 +5,8 @@ defmodule Unifex.Specs do
   For information on how to create such specs, see `Unifex.Specs.DSL` module.
   """
 
+  @type interface_t :: NIF | CNode
```

Comment:
- this is rather a generator, according to the new terminology
- should be `Unifex.CodeGenerator.t`

Reply:
nope, generator is a Elixir module, while interface is an atom you pass to the `interface` call in specs

Reply:
I thought we now have:
- `NIF`, `CNode` - modules, generators
- `:nif`, `:cnode` - interfaces
also, note that this type limits the space of possible generators to these two specified in advance, which I tried to avoid

Reply:
our API is that the user can pass `NIF` or `CNode` to the `interface` call in unifex `*.spec.exs`